addon/components/roadsigns-modal.js

```diff
@@ -78,6 +79,21 @@ export default class RoadsignRegulationCard extends Component {
 
   @action
   insertHtml(html) {
-    this.args.controller.executeCommand('insert-html', html);
+    const wrappedHtml = `
+      <div property="eli:has_part" typeof="besluit:Artikel" resource="http://data.lblod.info/artikels/${uuid()}">
+        <div property="eli:number" datatype="xsd:string">Artikel <span class="mark-highlight-manual">nummer</span></div>
+        <span style="display:none;" property="eli:language" resource="http://publications.europa.eu/resource/authority/language/NLD" typeof="skos:Concept">&nbsp;</span>
+        <div property="prov:value" datatype="xsd:string">
+          <span class="mark-highlight-manual">Voer inhoud in</span>
+        </div>
+        <div property="mobiliteit:heeftVerkeersmaatregel" typeof="mobiliteit:Mobiliteitsmaatregel" resource="http://data.lblod.info/mobiliteitsmaatregel/${uuid()}">
```

Comment:
given that mobiliteit is a specific vocabulary I think it's best the specify the prefix in this snippet